queue_services/entity-emailer/src/entity_emailer/message_tracker/tracker.py

```diff
@@ -61,6 +61,13 @@ def get_message_context_properties(queue_msg: nats.aio.client.Msg):
             identifier = email_msg.get('identifier', None)
             message_id = f'{etype}_{filing_id}'
             return create_message_context_properties(etype, message_id, None, identifier, False)
+
+        if etype == 'bc.registry.dissolution':
+            message_id = email_msg.get('id', None)
+            identifier = email_msg.get('identifier', None)
+            if involuntary_dissolution_stage_1_notification.is_processable(email_msg):
```

Comment:
Only implement a scenario for BC company that has overdue ARs for all email body template variations in designs.